src/relay/App.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import Setting from "./components/Setting";
+
+class App extends React.Component<Data> {
+  constructor(props: Data) {
+    super(props);
+  }
+  render(): React.ReactNode {
+    const elements = this.props.settings.map((setting) => {
+      return (
```

Comment:
You can drop the braces and return here.

Reply:
done